api/src/main/java/com/velocitypowered/api/command/VelocityBrigadierMessage.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2018 Velocity Contributors
+ *
+ * The Velocity API is licensed under the terms of the MIT License. For more details,
+ * reference the LICENSE file in the api top-level directory.
+ */
+
+package com.velocitypowered.api.command;
+
+import com.google.common.base.Preconditions;
+import com.mojang.brigadier.Message;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.ComponentLike;
+import net.kyori.adventure.text.serializer.plain.PlainTextComponentSerializer;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Represents an implementation of brigadier's {@link Message}, providing support for {@link
+ * Component} messages.
+ */
+public final class VelocityBrigadierMessage implements Message, ComponentLike {
+
+  private final Component message;
+
+  public VelocityBrigadierMessage(Component message) {
```

Comment:
Please expose this constructor as a `static` method, perhaps `VelocityBrigadierMessage.tooltip()`?

Reply:
Also make the constructor proper `private`.